src/resource-archive/index.ts

```diff
@@ -64,7 +70,7 @@ export class Watcher {
     await this.client.send('Fetch.enable');
   }
 
-  async idle(page: Page) {
+  async idle(page?: Page) {
```

Comment:
For now, the network-idle listening doesn't happen for Cypress. This will be [resolved](https://linear.app/chromaui/issue/AP-3843/cypress-handle-network-idleness) when I add that in a future PR